pkg/morph/client/constructor.go

```diff
@@ -160,8 +160,14 @@ func New(key *keys.PrivateKey, opts ...Option) (*Client, error) {
 		}
 
 		cli.endpoints = cfg.endpoints
+		for _, e := range cli.endpoints {
+			cli.client, act, err = cli.newCli(e)
+			if err != nil {
+				cli.logger.Warn("morph client connection", zap.String("endpoint", e), zap.Error(err))
```

Comment:
```suggestion
				cli.logger.Warn("NeoFS chain connection failure", zap.String("endpoint", e), zap.Error(err))
```
level is also doubtful, `Info` is enough imo

Reply:
the level is the same as it is done in [reconnection](https://github.com/nspcc-dev/neofs-node/blob/eba32a060dd0b1d919976d420b4ea740c268e9b0/pkg/morph/client/multi.go#L55)

> NeoFS chain

we use this client for mainchain too

Reply:
level - :ok_hand: 

> we use this client for mainchain too

then its extra not morph client, `Neo RPC connection failure`

Reply:
fixed